### DISCUSSION TITLE
Implementing a weldjoint in MuJoCO (from OpenSim to MuJoCo)

### DISCUSSION THREAD

**Discussion by GraceNiyo** [google-deepmind/mujoco#902] [Asking for Help] [answered]
I have an OpenSim model that I am converting to a MuJoCo model. In OpenSim a WeldJoint restricts any relative motion and maintains a fixed relationship between the bodies it connects. The bodies connected with a WeldJoint are permanently attached and cannot move relative to each other.  What type of joint and constraints could I use in MuJoCo to create a "WeldJoint" equivalent. I looked at MuJoCo documentation on the available joint types but none of the joint types works.

**Comment by yuvaltassa** [comment, marked as answer]
https://mujoco.readthedocs.io/en/latest/XMLreference.html#equality-weld

**Reply by GraceNiyo** [reply]
thank you so much.
